.github/workflows/build.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: canonical/craft-actions/rockcraft-pack@main
+        id: rockcraft
+
+      - name: Install Skopeo
+        run: |
+          sudo snap install skopeo --edge --devmode
+
+      - name: Import the image to Docker registry
+        run: |
+          sudo skopeo --insecure-policy copy oci-archive:${{ steps.rockcraft.outputs.rock }} docker-daemon:sdcore-gui:0.1
+
+      - name: Run the image
```

Comment:
Integration tests should not be part of `build`, they probably deserve a file of their own and be called on push.

Reply:
I've renamed the `build.yaml` to be `integration_tests.yaml`. I don't see the value of having the build logic in a separate file, let me know if that makes sense.